helm/ph-ee-engine/vouchers/templates/clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: pph-ee-vouchers-c-role
```

Comment:
we have changed this way of creating the role, please refer Ankita's PR for GOV 272